bridge/contracts/nftbridge/SideNFTToken.sol

```diff
@@ -11,12 +11,12 @@ contract SideNFTToken is ISideNFTToken, ERC721 {
     using Address for address;
     using SafeMath for uint256;
 
-    mapping(uint256 => uint256) _nonces;
+    mapping(uint256 => uint256) public _nonces;
 
     address public minter;
 
     // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2612.md
-    bytes32 public DOMAIN_SEPARATOR;
+    bytes32 public domainSeparator;
```

Comment:
In the other DOMAIN_SEPARATOR we used sanke case. We shuld choose one style

Reply:
Definitely, the lint says that to be on SNAK_CASE it should be a constant, but we set the domain separator at the constructor, so it can't be a constant 

In other files `Bridge.sol` and `SideToken.sol` I couldn't change to `domainSeparator`, because it's going to change the number of bytes of the variable name in contracts already deployed so at them I just added // solhint-disable-line var-name-mixedcase

Reply:
the number of bytes does not depend on the variable name but on the type. For example address is 20 bytes, uint256 is 32 bytes, bytes32 is 32 bytes.
So you could change a uint256 for a bytes32 as they are both 32 bytes, but you couldn't change it for an address as they are 20 bytes